components/backToStartVerticalButton/BackToStartVerticalButton.js

```diff
@@ -0,0 +1,42 @@
+import {
+  Button,
+  ButtonText,
+  UnfilledButtonImage,
+  FilledButtonImage,
+} from './BackToStartVerticalButton.styles';
+import { icons } from '../../style/icons';
+import React, { useState } from 'react';
+import { breakpointsObj } from '../../lib/responsive';
+import MediaQuery from 'react-responsive';
+export default function BackToStartVerticalButton({ onClick }) {
```

Comment:
formatting: please add empty line after imports (please do for all components, this helps w visibility